test/data/StorageProof.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "rlpParentNodes": "0xf8f8f8b18080a011eb05caf5fa62e9b0fffc9118cf6c7a6f10870db11d837223bf585fc7283b2c80a0be65b7596c589a734b53d15e0cc9f13eae2083bf0173b1c2baf1534d9273882d8080808080a0997f2ab256882c505e8887d9f7622a18a17dc5727a0e9d04d314e59d482a508780a088976b814f3477b2d25aa3992c31abc4a4299557a6e94ca7786a440b15a07f1e80a04b140101c7c54b2c4d69e4cf35a353d7d969e439c48fe7388eaec325f1bfe6578080f843a0390decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563a1a03132333400000000000000000000000000000000000000000000000000000008",
+    "root": "0x70b4172eb30c495bf20b5b12224cd2380fccdd7ffa2292416b9dbdfc8511585d",
+    "value": "0xa03132333400000000000000000000000000000000000000000000000000000008",
+    "path": "0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563",
+    "storageIndex": "00"
+  },
+  {
+    "rlpParentNodes": "0xf8f8f8b18080a011eb05caf5fa62e9b0fffc9118cf6c7a6f10870db11d837223bf585fc7283b2c80a0be65b7596c589a734b53d15e0cc9f13eae2083bf0173b1c2baf1534d9273882d8080808080a0997f2ab256882c505e8887d9f7622a18a17dc5727a0e9d04d314e59d482a508780a088976b814f3477b2d25aa3992c31abc4a4299557a6e94ca7786a440b15a07f1e80a04b140101c7c54b2c4d69e4cf35a353d7d969e439c48fe7388eaec325f1bfe6578080f843a03950028b2db7b1add5e94e63cac32ed35a09df16f7765865b09d4d973d51bb04a1a03536373800000000000000000000000000000000000000000000000000000008",
+    "root": "0x70b4172eb30c495bf20b5b12224cd2380fccdd7ffa2292416b9dbdfc8511585d",
+    "value": "0xa03536373800000000000000000000000000000000000000000000000000000008",
+    "path": "0xe950028b2db7b1add5e94e63cac32ed35a09df16f7765865b09d4d973d51bb04",
+    "key": "0x14bb2bf372bbfc1de82d7a80510e8bf9c0735e1982c822f370f0882fc1d4f607",
+    "storageIndex": "01"
+  }
+]
```

Comment:
We can merge AccountProof.json and StorageProof.json to a single file proof.json to avoid any duplication.